gitrevise/merge.py

```diff
@@ -29,7 +29,11 @@ class MergeConflict(Exception):
     pass
 
 
-def rebase(commit: Commit, new_parent: Optional[Commit]) -> Commit:
+def rebase(
+    commit: Commit,
+    new_parent: Optional[Commit],
+    known_state: Optional[Commit],
```

Comment:
My branch stacking tool uses this function. Could you maybe add ` = None` here to make some upgrade scenarios smoother?

Reply:
Sure. I see it's used a bit in this codebase, and I could remove some of the None arguments I added.